rdr_service/offline/sync_consent_files.py

```diff
@@ -15,13 +15,14 @@
 from rdr_service.api_util import copy_cloud_file, download_cloud_file, get_blob, list_blobs
 from rdr_service.dao import database_factory
 from rdr_service.services.gcp_utils import gcp_cp
+from rdr_service.storage import GoogleCloudStorageProvider
 
 SOURCE_BUCKET = {
     "vibrent": "ptc-uploads-all-of-us-rdr-prod",
     "careevolution": "ce-uploads-all-of-us-rdr-prod"
 }
 DEFAULT_GOOGLE_GROUP = "no-site-assigned"
-TEMP_CONSENTS_PATH = "./temp_consents"
+TEMP_CONSENTS_PATH = "/tmp/temp_consents"
 
 ParticipantData = collections.namedtuple("ParticipantData", ("participant_id", "origin_id", "google_group", "org_id"))
 
```

Comment:
We probably should not hard code the temp directory path here.  You can use the tempfile.gettempdir() function to get the current temp directory.  https://docs.python.org/3.7/library/tempfile.html#tempfile.gettempdir

Reply:
Updating now. Thanks, I didn't know that existed.